hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -44,18 +47,101 @@ class HassioIngressView extends LitElement {
 
   private _sessionKeepAlive?: number;
 
+  @state() private _resolveIngressURL: {
```

Comment:
Since this is an object, it will not cause a re-render when a prop is updated

Reply:
That's fine
```suggestion
  private _resolveIngressURL: {
```